src/garage/tf/policies/gaussian_mlp_policy2.py

```diff
@@ -0,0 +1,271 @@
+                 output_w_init=tf.glorot_uniform_initializer(),
+                 output_b_init=tf.zeros_initializer(),
+                 learn_std=True,
+                 adaptive_std=False,
+                 std_share_network=False,
+                 init_std=1.0,
+                 min_std=1e-6,
+                 max_std=None,
+                 std_hidden_sizes=(32, 32),
+                 std_hidden_nonlinearity=tf.nn.tanh,
+                 std_output_nonlinearity=None,
+                 std_parameterization='exp',
+                 layer_normalization=False):
+        if not isinstance(env_spec.action_space, akro.Box):
+            raise ValueError('GaussianMLPPolicy only works with '
+                             'akro.Box action space, but not {}'.format(
+                                 env_spec.action_space))
+        super().__init__(name, env_spec)
+        self.obs_dim = env_spec.observation_space.flat_dim
+        self.action_dim = env_spec.action_space.flat_dim
+
+        self._hidden_sizes = hidden_sizes
+        self._hidden_nonlinearity = hidden_nonlinearity
+        self._hidden_w_init = hidden_w_init
+        self._hidden_b_init = hidden_b_init
+        self._output_nonlinearity = output_nonlinearity
+        self._output_w_init = output_w_init
+        self._output_b_init = output_b_init
+        self._learn_std = learn_std
+        self._adaptive_std = adaptive_std
+        self._std_share_network = std_share_network
+        self._init_std = init_std
+        self._min_std = min_std
+        self._max_std = max_std
+        self._std_hidden_sizes = std_hidden_sizes
+        self._std_hidden_nonlinearity = std_hidden_nonlinearity
+        self._std_output_nonlinearity = std_output_nonlinearity
+        self._std_parameterization = std_parameterization
+        self._layer_normalization = layer_normalization
+
+        self.model = GaussianMLPModel2(
+            output_dim=self.action_dim,
+            hidden_sizes=hidden_sizes,
+            hidden_nonlinearity=hidden_nonlinearity,
+            hidden_w_init=hidden_w_init,
+            hidden_b_init=hidden_b_init,
+            output_nonlinearity=output_nonlinearity,
+            output_w_init=output_w_init,
+            output_b_init=output_b_init,
+            learn_std=learn_std,
+            adaptive_std=adaptive_std,
+            std_share_network=std_share_network,
+            init_std=init_std,
+            min_std=min_std,
+            max_std=max_std,
+            std_hidden_sizes=std_hidden_sizes,
+            std_hidden_nonlinearity=std_hidden_nonlinearity,
+            std_output_nonlinearity=std_output_nonlinearity,
+            std_parameterization=std_parameterization,
+            layer_normalization=layer_normalization,
+            name='GaussianMLPModel')
+
+    # pylint: disable=attribute-defined-outside-init
+    def build(self, state_input, name=None):
+        """Build model.
+
+        Args:
+          state_input (tf.Tensor) : State input.
```

Comment:
odd spacing -- the form is `state_input (tf.Tensor): State input.`

Reply:
fixed